.eslintrc.js

```diff
@@ -64,7 +65,7 @@ module.exports = {
       },
     },
     {
-      files: ['**/fixtures/handlers_*/**/*.js'],
+      files: ['**/fixtures/handlers_*/**/*.js', '**/fixtures/*es_module*/**/*.js'],
```

Comment:
if someone has a better suggestion for this regex, I'm all 👂 s

Reply:
This looks good to me :+1: 
In the future, we'll move to ES modules, where we'll be able to remove this.